website/source/docs/guides/monitoring-telegraf.html.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: "docs"
+page_title: "Monitoring Consul with Telegraf"
+sidebar_current: "docs-guides-monitoring-telegraf"
+description: |-
+  Best practice approaches for monitoring a production Consul cluster with Telegraf.
+---
+
+# Monitoring Consul with Telegraf
+
+Consul has a number of metrics available through http endpoints. Measuring these metrics is useful for both health checking and debugging of cluster issues.
+
+There are number of monitoring tools and options, but for the purposes of this guide we are going to use the [telegraf_plugin][].
+
+The core points will be the same if other tools are used, but the names of the metrics may be different.
+
+## Configuring Telegraf
+
+# Installing Telegraf
+
+Installing Telegraf is straightforward on most Linux distributions. We recommend following the [official Telegraf installation documentation][telegraf-install].
+
+# Configuring Telegraf
+
+Besides acting as a statsd agent, Telegraf can collect additional metrics of its own. Telegraf itself ships
```

Comment:
maybe want to clarify "about the host that the Consul agent is running on"

Reply:
Added.